packages/components/src/button/style.scss

```diff
@@ -212,7 +212,7 @@
 			outline: none;
 		}
 
-		&:active:focus:enabled {
+		&:not([aria-disabled="true"]):active:focus {
 			box-shadow: none;
 		}
```

Comment:
The styles changes make me a bit nervous :P because we have some many button styles ovverrides in the UI that I know this can trigger specificity issues and should be tested properly. I'm thinking in these situations